### PR TITLE
R0.7

### PR DESCRIPTION
**Pull request by vedant5681**
I need this code to learn tensorflow so please allow request

### CODE REVIEW

File: tensorflow/g3doc/how_tos/tool_developers/index.md [hunk 1]
```diff
@@ -28,17 +28,15 @@ by calling `as_graph_def()`, which returns a `GraphDef` object.
 
 The GraphDef class is an object created by the ProtoBuf library from the
 definition in
-[tensorflow/core/framework/graph.proto](https://github.com/tensorflow/tensorflow
-/blob/master/tensorflow/core/framework/graph.proto). The protobuf tools parse
+[tensorflow/core/framework/graph.proto](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/graph.proto). The protobuf tools parse
```
Comment (martinwicke) [inline tensorflow/g3doc/how_tos/tool_developers/index.md]: Clearly this is not this PRs fault, but the link should point to tensorflow.org/code/tensorflow/core/framework/graph.proto

File: tensorflow/g3doc/how_tos/tool_developers/index.md [hunk 2]
```diff
@@ -28,17 +28,15 @@ by calling `as_graph_def()`, which returns a `GraphDef` object.
...
 this text file, and generate the code to load, store, and manipulate graph
 definitions. If you see a standalone TensorFlow file representing a model, it's
 likely to contain a serialized version of one of these `GraphDef` objects
 saved out by the protobuf code.
 
 This generated code is used to save and load the GraphDef files from disk. A
 good example to look at as we dig into this is
-[graph_metrics.py](https://github.com/tensorflow/tensorflow/blob/master/tensorfl
-ow/python/tools/graph_metrics.py). This Python script takes a saved graph
+[graph_metrics.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/graph_metrics.py). This Python script takes a saved graph
```
Comment (martinwicke) [inline tensorflow/g3doc/how_tos/tool_developers/index.md]: Same here

File: tensorflow/g3doc/how_tos/tool_developers/index.md [hunk 3]
```diff
@@ -115,8 +111,7 @@ This defines what operation to run, for example `"Add"`, `"MatMul"`, or
 `"Conv2D"`. When a graph is run, this op name is looked up in a registry to
 find an implementation. The registry is populated by calls to the
 `REGISTER_OP()` macro, like those in
-[tensorflow/core/ops/nn_ops.cc](https://github.com/tensorflow/tensorflow/blob/ma
-ster/tensorflow/core/ops/nn_ops.cc).
+[tensorflow/core/ops/nn_ops.cc](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/nn_ops.cc).
```
Comment (martinwicke) [inline tensorflow/g3doc/how_tos/tool_developers/index.md]: I suspect all these are wrong in master too.

**Comment by vrv** [comment]
Just to be clear, these are commits to 0.7 that you're trying to forward port to master?

**Comment by martinwicke** [comment]
Ah. Well we should fix some of these properly then.

**Pull request closed without merging**